tools/cardano-sl-tools.cabal

```diff
@@ -261,8 +243,7 @@ executable cardano-swagger
 
   if flag(for-installer)
     buildable: False
-
-  if !flag(with-wallet) || !flag(with-web)
+  else
     buildable: False
```

Comment:
@gromakovsky Why keygen? This changes are related to `cardano-swagger` executable and not to `cardano-dht-keygen`.